plugins/certs/subcommands/add

```diff
@@ -71,6 +71,7 @@ cmd-certs-set() {
   fi
 
   mkdir -p "$APP_SSL_PATH"
+  rm "$APP_SSL_PATH/server.crt" "$APP_SSL_PATH/server.key" 2>/dev/null
```

Comment:
Maybe a comment explaining that this is necessary for correctly handling symlinks would be useful here?
Also, instead of  `2>/dev/null`, what about `rm -f`?

Reply:
@ltalirz I would assume the why would be due to the commit/change.. else the whole code base would be full of comments as to the why something is done?

I will change the rm command as proposed.. no idea why I did it like that, must have been late :sheep:

Reply:
Let me know if you still want the comment.

Reply:
Thanks @AubreyHewes , I just thought the reason for this `rm` is probably not obvious from looking at the code (which means someone might might delete it in the future).
Anyhow, when or not to put comments is a matter of taste, so just leave it as-is - @josegonzalez will anyhow give another review.

Reply:
@ltalirz I understand and agree with the reasoning.. and I know that it could alleviate possible future problems (i.e. someone removing it as they think it is unnecessary as you describe).. I actually am for a little comment on the why. Though due to the very little amounts of explanatory comments in the code I was reluctant.. as it seems not to be the way in this codebase.. A change with the comment was already prepped ;-)

Reply:
I normally try making the commit message verbose to explain why I did a thing. A comment in code is more likely to diverge over time from the code underneath, whereas you can write whole narratives within commit messages.